Can't Access http://localhost:3000/

i try use this boilerplate but i can't access http://localhost:3000/ or http://boilerplate.test:3000/ just loading...

![image](https://user-images.githubusercontent.com/14294482/67746102-5adcda00-fa58-11e9-9eb4-f1e8b2a7fbb6.png)

Same here,  I got this message when doing `npm start`
``` ~/locomotive-boilerplate-master/node_modules/esbuild/lib/main.js:236:12: error: Invalid option in build() call: "errorLimit"
    236 │       throw new Error(`Invalid option ${where}: "${key}"`); ```

--- a/build/watch.js
+++ b/build/watch.js
@@ -8,11 +8,20 @@ import paths from '../mconfig.json';
 import bs from 'browser-sync';
 
 // Start the Browsersync server
-bs.init({
-    proxy: paths.url,
+let bsConfig = {
     open: false,
     notify: false
-});
+};
+
+if (typeof paths.proxy === 'string' && paths.proxy.length > 0) {
+    // Use proxy
+    bsConfig.proxy = paths.proxy;
+} else {
+    // Use base directory
+    bsConfig.server = { baseDir: paths.dest };
+}
+
+bs.init(bsConfig);
 
 // Build scripts, compile styles, concat vendors and generate the svgs sprite on first hit
 buildScripts();
@@ -31,7 +40,7 @@ bs.watch(
     ]
 ).on('change', bs.reload);
 
-// Watch scripts 
+// Watch scripts
 bs.watch(
     [
         paths.scripts.src + '**/*.js'

--- a/mconfig.json
+++ b/mconfig.json
@@ -1,5 +1,5 @@
 {
-    "url": "locomotive-boilerplate.test",
+    "proxy": "",
     "src": "./assets/",
     "dest": "./www/",
     "build": "./build/",

